src/CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ target_link_libraries(onnx-mlir
         ${MLIRLibs}
         ${CMAKE_DL_LIBS})
 
+if (INCLUDE_ONNX_ML)
```

Comment:
Always build it.

Reply:
By default, we build the onnx ml. At a meeting, someone asked for the support that we choose not to build onnx ml.  I have no preference over this issue. @tjingrant  @AlexandreEichenberger What's your opinion?